.github/workflows/create_tag.yml

```diff
@@ -395,13 +414,22 @@ jobs:
             return
             
           REPO = os.environ.get('REPO')
+          OWNER = os.environ.get('OWNER')
+          GITHUB_TOKEN = os.environ.get('GITHUB_TOKEN')
           COMMIT_HASH = os.environ.get('COMMIT_HASH')
+          CUSTOM_VERSION = os.environ.get('CUSTOM_VERSION')
```

Comment:
Use the variable name as `USE_CUSTOM_VERSION` which indicates it is a boolean. Name `CUSTOM_VERSION` indicates it is a version